### PR TITLE
[Feat] notes functionality

### PR DESCRIPTION
**Pull request by aryan-13**
Added:
- Add notes
- remove notes

image

### CODE REVIEW

**Review by anshikaa12 — APPROVED** [review]
code looks clean
it looks well structured
you can merge it

File: src/Components/MainSection/Notes/Notes.css [hunk 1]
```diff
@@ -34,21 +37,33 @@
...
 	display: flex;
 	justify-content: center;
 	flex-wrap: wrap;
 }
 .note-component {
+	display: flex;
+	flex-direction: column;
 	border-radius: 5px;
-	border: 1px solid black;
+	/* border: 1px solid black; */
```
Comment (mnath30) [inline src/Components/MainSection/Notes/Notes.css]: Remove the commented css if not required

File: src/Components/MainSection/Notes/Notes.css [hunk 2]
```diff
@@ -34,21 +37,33 @@
...
+	justify-content: space-between;
 	font-size: 1.6rem;
 }
 .note-icon {
+	margin-left: auto;
 	align-self: flex-end;
 	cursor: pointer;
 }
+.note-text {
+	/* overflow-wrap: break-word; */
```
Comment (mnath30) [inline src/Components/MainSection/Notes/Notes.css]: Here as well

File: src/Components/MainSection/Notes/Notes.jsx [hunk 3]
```diff
@@ -10,19 +15,45 @@ function Notes() {
 					<textarea
 						placeholder="Take a note..."
 						className="notes-input"
+						value={currNote}
+						onChange={(e) => {
+							if (e.target.value.length > 180) {
```
Comment (mnath30) [inline src/Components/MainSection/Notes/Notes.jsx:20]: Really liked this part where the text input size handling is taken care of

**Review by mnath30 — APPROVED** [review]
Code is clean 
Liked the part where the empty and character limit condition was handled 
Ui look so good 😃

**Pull request merged**